datumaro/plugins/data_formats/ava/ava.py

```diff
@@ -0,0 +1,222 @@
+    PROPOSAL_EXT = ".pkl"
+
+
+class AvaBase(SubsetBase):
+    def __init__(self, path):
+        if not osp.isfile(path):
+            raise DatasetImportError(f"Can't find JSON file at '{path}'")
+        self._path = path
+
+        subset = (
+            osp.splitext(osp.basename(path))[0]
+            .replace(AvaPath.ANNOTATION_PREFIX, "")
+            .replace(AvaPath.ANNOTATION_VERSION, "")
+        )
+        super().__init__(subset=subset)
+
+        if path.endswith(osp.join(AvaPath.ANNOTATION_DIR, osp.basename(path))):
+            self._rootpath = path.rsplit(AvaPath.ANNOTATION_DIR, maxsplit=1)[0]
+        else:
+            raise DatasetImportError(
+                f"Annotation path ({path}) should be under the directory which is named {AvaPath.ANNOTATION_DIR}. "
+                "If not, Datumaro fails to find the root path for this dataset."
+            )
+
+        if self._rootpath and osp.isdir(osp.join(self._rootpath, AvaPath.IMAGE_DIR)):
+            self._images_dir = osp.join(self._rootpath, AvaPath.IMAGE_DIR)
+        else:
+            raise DatasetImportError(
+                f"Root path ({self._rootpath}) should contain the directory which is named {AvaPath.IMAGE_DIR}. "
+                "If not, Datumaro fails to find the image directory path."
+            )
+
+        self._infos = self._load_infos(osp.dirname(path))
+
+        category_path = osp.join(self._rootpath, AvaPath.ANNOTATION_DIR, AvaPath.LABEL_LIST)
+        self._categories = self._load_categories(category_path)
+
+        self._items = self._load_items(path)
+
+    def _load_infos(self, path):
+        infos = {}
+        for file in os.listdir(path):
+            if file.endswith(AvaPath.PROPOSAL_EXT):
+                name = file.split(".")[0].split("_")[-1]
+                infos[name + "_proposals"] = file
+
+        return infos
+
+    def _load_categories(self, category_path):
+        if not osp.exists(category_path):
+            raise DatasetImportError(
+                f"Label lists cannot be found in ({category_path}). "
+                "If not, Datumaro fails to import AVA action dataset."
+            )
+
+        with open(category_path, "r") as f:
+            pbtxt_data = f.read()
+
+        label_list = ava_label_pb2.LabelList()
+        text_format.Parse(pbtxt_data, label_list)
+
+        categories = LabelCategories()
+
+        # dummy class for id 0 for ava data
+        if label_list.label[0].label_id != 0:
+            categories.add("no action")
+        for node in label_list.label:
+            categories.add(node.name)
+
+        return {AnnotationType.label: categories}
+
+    def _load_items(self, ann_file):
+        items = {}
+        with open(ann_file, "r", encoding="utf-8") as f:
+            csvreader = csv.reader(f)
+            datas = list(csvreader)
+            for data in datas:
+                video_id = data[0]
+                timestamp = data[1]
+
+                item_id = video_id + "/" + timestamp
```

Comment:
Do we have to keep this separator, "/"? Is it possible to change it to "_"?

Reply:
I felt that it is more preferred to use "/" for item_id of which including mixed information. Please see belows.
https://github.com/openvinotoolkit/datumaro/blob/f1c467a967e1b88f545817007f859e0015a04b44/datumaro/plugins/data_formats/icdar/base.py#L102
https://github.com/openvinotoolkit/datumaro/blob/f1c467a967e1b88f545817007f859e0015a04b44/datumaro/plugins/data_formats/lfw.py#L270
https://github.com/openvinotoolkit/datumaro/blob/f1c467a967e1b88f545817007f859e0015a04b44/datumaro/plugins/data_formats/widerface.py#L93